source/portal/v11Client.js

```diff
@@ -6,6 +6,7 @@
 
```

Comment:
v11client.js and v10client.js won't be used in current code base.
1.1 and 1.2 requests will be adapted in client.js

Reply:
I'll remove this one. It has been modified in an earlier change. `client.js` is also updated during rebasing.